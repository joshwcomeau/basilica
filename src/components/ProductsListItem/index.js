// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import './index.scss';


const ProductsListItem = () => {
  const classes = classNames(['products-list-item']);

  return (
    <div className={classes}>
      Your Component Here :)
    </div>
  );
};

ProductsListItem.propTypes = {

};

ProductsListItem.defaultProps = {

};

export default ProductsListItem;
