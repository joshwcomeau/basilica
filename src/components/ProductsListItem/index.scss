@import '../../variables';

.products-list-item {


  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
