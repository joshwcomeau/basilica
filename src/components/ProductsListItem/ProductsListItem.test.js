/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import ProductsListItem from './index';

describe('ProductsListItem', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ProductsListItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
