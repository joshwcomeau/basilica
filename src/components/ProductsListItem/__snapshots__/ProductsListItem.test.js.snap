exports[`ProductsListItem renders correctly 1`] = `
<div
  className="products-list-item">
  Your Component Here :)
</div>
`;
